src/room.ts

```diff
@@ -353,15 +347,15 @@ export class Room extends EventEmitter implements Sayable {
 
   // should be deprecated
   public nick(contact: Contact): string {
-    log.warn('Room', 'nick(Contact) DEPRECATED, use name(Contact) instead.')
+    log.warn('Room', 'nick(Contact) DEPRECATED, use alias(Contact) instead.')
     return this.alias(contact)
   }
 
   public alias(contact: Contact): string {
-    if (!this.obj || !this.obj.nameMap) {
+    if (!this.obj) {
       return ''
     }
-    return this.obj.roomAliasMap[contact.id] || this.obj.nameMap[contact.id]
+    return this.obj.aliasMap[contact.id]
```

Comment:
if no `aliasMap[contact.id]`, here'll return a `undefined`, which is not expected?